iac-extensions/docker/src/test/resources/checks/RetrieveRemoteResourcesCheck/curl.dockerfile

```diff
@@ -428,7 +398,29 @@ RUN curl -L https://example.com/resource -s --output output.txt -k -H "X-Auth-To
 RUN curl --header "X-Auth-Token 123" https://example.com/resource --output output.txt -L
 RUN curl -L https://example.com/resource -s --output output.txt --header "X-Auth-Token 123" -k
 
-# Compliant save in /dev/null
+# Compliant: ADD doesn’t support HTTP request options =======
+
+RUN curl -o output.txt https://example.com/resource --data "param1=value1"
+RUN curl -o output.txt https://example.com/resource -d "param1=value1"
+RUN curl -o output.txt https://example.com/resource --data-ascii @file
+RUN curl -o output.txt https://example.com/resource --data-binary @file
+RUN curl -o output.txt https://example.com/resource --data-raw "param1=value1"
+RUN curl -o output.txt https://example.com/resource --data-urlencode param1=value1
+RUN curl -o output.txt https://example.com/resource --form param1=value1
+RUN curl -o output.txt https://example.com/resource -F param1=value1
+RUN curl -o output.txt https://example.com/resource --form-escape -F 'param1=value1'
+RUN curl -o output.txt https://example.com/resource --form-string "param1=value1"
+RUN curl -o output.txt https://example.com/resource --header "Content-Type: application/json"
+RUN curl -o output.txt https://example.com/resource -H "Content-Type: application/json"
+RUN curl -o output.txt https://example.com/resource --json '{"param1": "value1"}'
+RUN curl -o output.txt https://example.com/resource --referer "https://example.com/referer"
+RUN curl -o output.txt https://example.com/resource -e "https://example.com/referer"
+RUN curl -o output.txt https://example.com/resource --request POST
+RUN curl -o output.txt https://example.com/resource -X POST
+RUN curl -o output.txt https://example.com/resource --user-agent "Mozilla/5.0"
+RUN curl -o output.txt https://example.com/resource -A "Mozilla/5.0"
```

Comment:
I would expect that those examples are Noncompliant, because "ADD doesn’t support HTTP request options" (line 401)

Reply:
Consider to add/modify some of those examples to have "request options" not always at the end but before or after `-o output.txt` argument. It is for better coverage.

Reply:
Unsure about what you mean. To me these examples are compliant because ADD doesn't support those flags (like it doesn't support auth), so we can't actually raise an issue on these RUN commands; they can't be replaced by ADD commands as the issue would suggest.

Reply:
Sorry my bad, I confused myself. The selected examples are Compliant.
However, please add a few examples with different argument order, e.g.:
```
RUN curl --data "param1=value1" -o output.txt https://example.com/resource 
RUN curl -o output.txt --data "param1=value1" https://example.com/resource
RUN curl -o output.txt --data "param1=value1" https://example.com/resource --data "param2=value2"

```